python/deltalake/table.py

```diff
@@ -456,6 +456,27 @@ def vacuum(
             enforce_retention_duration,
         )
 
+    def update(
+        self,
+        updates: Dict[str, str],
+        predicate: Optional[str] = None,
+        writer_properties: Optional[Dict[str, int]] = None,
+        strict_cast: bool = True,
+    ) -> Dict[str, Any]:
+        """UPDATE records in the Delta Table that matches an optional predicate.
+
+        :param updates: what values to update
+        :param predicate: a logical expression, defaults to None
+        :writer_properties: Pass writer properties to the Rust parquet writer, see options https://arrow.apache.org/rust/parquet/file/properties/struct.WriterProperties.html
+        :strict_cast: specify if data types need to be casted strictly or not :default = True
+        :return: the metrics from delete
```

Comment:
Perhaps you could say what fields are in the metrics? Or at least the important ones?